pkgs/by-name/ar/argc/package.nix

```
@@ -0,0 +1,64 @@
+{
+  lib,
+  rustPlatform,
+  stdenv, # Determin platforms.
+  fetchFromGitHub,
+  nixpkgsFun, # Re-evaluate Nixpkgs.
+  argc, # Get native argc for shell completion generation during cross-compilation.
```

```suggestion
  stdenv,
  fetchFromGitHub,
  nixpkgsFun,
  argc,
```
This is all clear from the package definition in the rest of the file.